.devcontainer/Dockerfile

```diff
@@ -13,4 +14,12 @@ RUN apt update -y && apt upgrade -y \
     libgtest-dev \
     # Dependency for intel-mkl
     libomp-dev \
-  && rm -rf /var/lib/apt/lists/*
+    # Dependency for clangd-tidy
+    python3 \
+    python3-pip \
+    python3-venv \
+  && rm -rf /var/lib/apt/lists/* 
+  
+RUN python3 -m venv /.venv \
+  && PATH=/.venv/bin:$PATH pip install clangd-tidy \
```

Comment:
Where is there documentation for this package? The only doc I can find is https://pypi.org/project/clang-tidy/ but not clangd-tidy.

Reply:
It's here: https://github.com/lljbash/clangd-tidy

I've added a note in the Dockerfile explaining this choice.